backend/entityservice/init-db-schema.sql

```diff
@@ -85,6 +85,12 @@ RETURNS bool AS $$
   SELECT $1.state = 'completed'
 $$ STABLE LANGUAGE SQL;
 
+CREATE TYPE UPLOADEDSTATE AS ENUM (
```

Comment:
wow, now we have `UPLOADEDSTATE` and `UPLOADSTATE`. This is very confusing.
After looking at those for a bit, I think we should rename `UPLOADSTATE` to `PROCESSEDSTATE`.
It would also be nice to add a bit of an explanation what all those states are for.
In my view, 
UPLOADSTATE describes the state of the processing of the uploaded clks. This is only set *after* CLKs have been uploaded. 'pending' means we are waiting for some processing to be done, 'ready' means done and 'error' if something fails.
UPLOADEDSTATE describes the state of the CLK upload.

Reply:
That was indeed a pickle. I will add comments on the `UPLOADEDSTATE`.
I didn't want to touch the `UPLOADSTATE` as I wasn't sure what it was made for (as you mentioned, this is not well documented). But I'll also add some documentation around it and rename it. In which case I will also rename `UPLOADEDSTATE` to `UPLOADSTATE` which sounds better :)

Reply:
I'll finally keep `UPLOADEDSTATE` instead of renaming it because the value in the table is `uploaded`. But renamed the `UPLOADSTATE` to `PROCESSEDSTATE`, adding some documentation.